srcpkgs/telegram-desktop/template

```diff
@@ -12,20 +13,26 @@ configure_args="-DTDESKTOP_API_ID=209235
  -DQT_HOST_PATH=/usr"
 hostmakedepends="pkg-config qt6-base python3 extra-cmake-modules
  qt6-wayland-tools wayland-devel protobuf"
-makedepends="alsa-lib-devel ffmpeg-devel libdbusmenu-glib-devel libopenal-devel
- minizip-devel opus-devel xxHash-devel pulseaudio-devel range-v3
- qt6-svg-devel libva-devel rapidjson liblz4-devel liblzma-devel gtk+3-devel
- MesaLib-devel qt6-wayland-devel qt6-qt5compat-devel libatomic-devel
- xcb-util-keysyms-devel $(vopt_if spellcheck hunspell-devel) protobuf-devel
- glibmm2.68-devel tg_owt webkit2gtk-devel rnnoise-devel jemalloc-devel qt6-declarative-devel"
+makedepends="alsa-lib-devel boost-devel clang ffmpeg-devel fmt-devel
```

Comment:
This shouldn't be reordered just for the sake of reordering. Why is there clang in makedepends?

Reply:
The build runs on clang now, instead of gcc. Is there another way to do this?

Reply:
Not sure how that would work, the cross builds would be broken if it actually executes `clang` since its in `makedepends` and not `hostmakedepends` so its installed for the target architecture and can't be executed.

Reply:
So if its really required then it belongs in `hostmakedepends`, but since it seems like its not actually used, I'm not sure?